### PR TITLE
Docs/mm 218/documentation changes

### PR DESCRIPTION
**Pull request by monitz87**

### CODE REVIEW

File: src/api/entities/Proposal/types.ts [hunk 1]
```diff
@@ -28,12 +28,6 @@ export interface Metadata {
   totalVotes: number;
 }
 
-export interface Proposal {
-  call: Call;
-  metadata: Metadata;
-  state: ProposalState;
-}
-
```
Comment (VictorVicente) [inline src/api/entities/Proposal/types.ts:36]: Why is this deleted?

Comment (shuffledex) [reply src/api/entities/Proposal/types.ts:36]: I added it at the beginning of the Proposal entity but we never used it
